mlir/examples/standalone/really_alone/mlir_standreallyalone/dialects/_ods_common.py

```diff
@@ -0,0 +1,307 @@
+#  Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+#  See https://llvm.org/LICENSE.txt for license information.
+#  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+from typing import (
+    List as _List,
+    Optional as _Optional,
+    Sequence as _Sequence,
+    Tuple as _Tuple,
+    Type as _Type,
+    Union as _Union,
+)
+
+from mlir._mlir_libs import _mlir as _cext
+from mlir.ir import (
```

Comment:
bcause `_standalone_ops_gen.py` uses helpers from here and they need to refer to the upstream extension instead of the downstream extension (which won't exist).